src/theme/MinimalRequirements.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import RubyIcon from '@site/static/img/ruby.png';
+import RailsIcon from '@site/static/img/rails.png';
+
+export default function MinimalRequirements(props) {
+  return (
+    <div class="minimal_requirements">
+      <p>Minimal requirements:</p>
+      <dl>
+        <dt>
+          <img src="/img/ruby.png" alt="Ruby" />
+        </dt>
+        <dd>v{ props.ruby }</dd>
+        <dt>
+          <img src="/img/rails.png" alt="Rails" />
```

Comment:
SVGs would be better here, some links below:

https://commons.wikimedia.org/wiki/File:Ruby_logo.svg
https://rubyonrails.org/assets/images/logo.svg
https://commons.wikimedia.org/wiki/File:Ruby_On_Rails_Logo.svg
https://pressupinc.com/wp-content/uploads/2014/11/Ruby_on_Rails.svg_.png

Reply:
Good call. Updated.